js/deleteTask.js

```diff
@@ -0,0 +1,11 @@
+//import { taskList } from "/js/addToTask.js";
+import { CLICK_EVENT, DELETE, UL_LIST } from "/js/constants.js";
+
+export default function removeTodos(element, taskList) {
+  let index = taskList.findIndex((item) => {
+    return item.id == element.id;
+  });
```

Comment:
use `const` over let since we're not updating the variable

```suggestion
  const index = taskList.findIndex(({ id }) => id == element.id);
```